brianlong/www.validators.app
# 183532936 gather all stake accounts

#### What's this PR do?
- script for gathering stake accounts for explorer

#### How should this be manually tested?
- run tests
- make sure You have any EpochWallClocks
- run `rails r script/gather_stake_accounts.rb`
- alternatively you can run the service in ruby console `GatherExplorerStakeAccountsService.new(network: "mainnet", config_urls:, current_epoch: nil, demo: true, stake_accounts: []).call`

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183532936)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [Y] Does this PR add new dependencies?
